### ISSUE TITLE
DS data page: Update body text

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2821]
### Story

As a user, I want the abbreviation "DS" to be expanded the first time it appears on a page, so I can understand its meaning without confusion and have a better user experience, especially if I am unfamiliar with technical jargon.

### Acceptance Criteria

- [ ] On the DS page change this line of body text `Enter the values given by your DNS provider for DS Data.` to` Enter the values given by your DNS provider for DS (Delegation Signer) Data.`

### Additional Context

Today, this fails WCAG SC 3.1.4 Abbreviations

### Issue Links

⏬ Caused by DS data page cancel button resets form #2700

**Comment by AnnaGingle** [comment]
This issue will be completed as a part of issue #2849

**Comment by katypies** [comment]
This was completed!
